ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -76,3 +76,15 @@ if [ -n "${FEATURE_GATES}" ]; then
 featureGates: ${FEATURE_GATES}
 EOF
 fi
+
+if [ ${ADDITIONAL_NTP_SERVERS,,} = "true" ]; then
+# 4.18 feature
+echo "Creating patch file to add Additional NTP servers"
+cat > "${SHARED_DIR}/ntpservers_patch_install_config.yaml" <<EOF
+platform:
+  baremetal:
+    additionalNTPServers:
+      - 10.11.160.238
```

Comment:
can we take this list as a comma-separated list of Hostnames/IPs from the cluster profile secret in the vault?

Reply:
The NTP server IPs used are the IPs of [clock.redhat.com](http://clock.redhat.com/). I dont know if we need to add it to vault. I can use the domain name and check for it in the ginko test.

Reply:
Can we then infer the IPs from that hostname with some command like `dig +short clock.redhat.com`?